sys/build.sh

```diff
@@ -140,7 +133,7 @@ if [ -z "${KEEP_PLUGINS_CFG}" ]; then
 fi
 unset DEPS
 pwd
-./configure ${CFGARG} --prefix="${PREFIX}" || exit 1
```

Comment:
why are you removing the ability to the user to specify a different prefix?

Reply:
WAT, I always run with --prefix, don't remove this